rust/src/smb/smb.rs

```diff
@@ -1101,6 +1109,21 @@ impl SMBState {
      * was received. */
     fn post_gap_housekeeping(&mut self, dir: Direction)
     {
+        let timeoutts = self.ts - 30;
+        if self.ts_ssn_gap || self.tc_ssn_gap {
+            let _pre_size = self.guid2name_map.len();
+            self.guid2name_map.retain(|_,(_p,ts)| {
```

Comment:
Is this iterating all the entries in the (unbounded) hashmap ?..

Reply:
I meant this looks bad :
If I put 65k elements in this unbounded hash maps, and then have 65k mini gaps (in less than 30 seconds), You get 4G iterations

I think there should be bounds to the number of elements that can be added to this hashmap...

Reply:
Using the Lru experiment here #11786 we don't need to do anything anymore. We just set a max size.

Reply:
Cool :-)